### PR TITLE
Add dev.mufeed.Wordbook

### PR DESCRIPTION
**Pull request by mufeedali**
Intended to update and change app id for https://github.com/flathub/com.github.fushinari.Wordbook

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: dev.mufeed.Wordbook.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          "type": "archive",
+          "url": "https://github.com/espeak-ng/pcaudiolib/archive/1.2.tar.gz",
+          "sha256": "44b9d509b9eac40a0c61585f756d76a7b555f732e8b8ae4a501c8819c59c6619"
+        }
+      ]
+    },
+    {
+      "name": "espeak-ng",
+      "cleanup": [
+        "*.la"
```
Comment (bbhtt) [inline dev.mufeed.Wordbook.json:29]: Move this to a global cleanup. Add `*.a`, `*.la`, `/lib/pkgconfig`, `/include`

Comment (mufeedali) [reply dev.mufeed.Wordbook.json]: Hi, should be fixed now!

File: dev.mufeed.Wordbook.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      ]
+    },
+    {
+      "name": "wordbook",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/mufeedali/Wordbook.git",
```
[the remark this answers is not in the document]
Comment (mufeedali) [reply dev.mufeed.Wordbook.json:95]: Fixed!

**Comment by bbhtt** [comment]
bot, build dev.mufeed.Wordbook

**Comment by bbhtt** [comment]
> Intended to update and change app id for https://github.com/flathub/com.github.fushinari.Wordbook

You need to EOL rebase this one https://docs.flathub.org/docs/for-app-authors/maintanance#end-of-life. Once that is merged, open an issue in this repo to archive that.

Why was the name change necessary?

**Comment by mufeedali** [comment]
> You need to EOL rebase this one [docs.flathub.org/docs/for-app-authors/maintanance#end-of-life](https://docs.flathub.org/docs/for-app-authors/maintanance#end-of-life). Once that is merged, open an issue in this repo to archive that.

Wouldn't that have to be after this is added?

> Why was the name change necessary?

Multiple reasons, one is I'm moving away from my older username "fushinari". Another is that I want it to be a more permanent personal app id. mufeed.dev is my own site. I might move hosting away from GitHub as well in the future. Now seemed like the right time to make the switch.

**Comment by bbhtt** [comment]
> Wouldn't that have to be after this is added?

Yea

> Multiple reasons, one is I'm moving awa

Thanks.

**Pull request closed without merging**